src/components/icons/FontDownloadIcon.tsx

```diff
@@ -0,0 +1,11 @@
+import { SvgIcon, SvgIconProps } from '@material-ui/core';
+import React, { FC } from 'react';
+
+export const FontDownloadIcon: FC<SvgIconProps> = props => {
```

Comment:
what is the purpose of not using material icons?

Reply:
I'm adding custom material icons from https://fonts.google.com/icons, that are not presented in the Material-UI set. A 'FontDownload' icon exists in the material-ui but it is filled, unlike the one I've added.